services/agents/registry.go

```diff
@@ -841,9 +871,7 @@ func (r *Registry) CheckConnectionToService(ctx context.Context, q *reform.Queri
 		if err = q.Update(agent); err != nil {
 			return errors.Wrap(err, "failed to update table count")
 		}
-
 	case models.ExternalServiceType, models.HAProxyServiceType:
-		// TODO: handle check of exporter response format https://jira.percona.com/browse/PMM-5778
 		l.Debugf("CheckConnectionResponse: %+v.", resp)
```

Comment:
let's remove this line, we already print it in line 864 https://github.com/percona/pmm-managed/pull/670/files#diff-e0ef3ef905963456d8d5b9719c09ab6328c3a3db255dfe9da41e89b067dec5bdR864